core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -116,6 +118,9 @@
   /** Used to stop catching up when cancellation requested.  */
   private volatile boolean mStopCatchingUp = false;
 
+  private long mLastCheckPointTime = -1;
+  private long mEntriesSinceLastCheckPoint = 0;
```

Comment:
We might be able to reuse checkpoint stats stored in `UfsJournalCheckpointThread`.